From 2248df8bd1da1127a8d85418877f084af64de8ec Mon Sep 17 00:00:00 2001
From: Harry Youd <harry@harryyoud.co.uk>
Date: Sun, 19 Feb 2017 11:22:34 +0000
Subject: [PATCH 152/214] Last week in LineageOS (w/c 13th Feb 2017)

- aptx(hd)
- CAF rebase and r21
- DotSrc build slave
- Long-press power = torch
- Changelog viewer on updater page
- device roster

Change-Id: Ib5b477fa57a4e336f31ce387e9b94f1f3acc069a
---
 _posts/2017-02-20-Last-Week-in-LineageOS-2.md | 36 +++++++++++++++++++
 1 file changed, 36 insertions(+)
 create mode 100644 _posts/2017-02-20-Last-Week-in-LineageOS-2.md

diff --git a/_posts/2017-02-20-Last-Week-in-LineageOS-2.md b/_posts/2017-02-20-Last-Week-in-LineageOS-2.md
new file mode 100644
index 0000000..bc6e390
--- /dev/null
+++ b/_posts/2017-02-20-Last-Week-in-LineageOS-2.md
@@ -0,0 +1,36 @@
+---
+layout: post
+title: Last week in LineageOS
+category: blog
+excerpt: W/C 13th Feb 2017
+author: harryyoud
+---
+
+## Welcome to LineageOS' weekly review, where we go over changes in the past week
+
+### Major changes since 13th February 2017
+* AptX and AptXHD have been added to supported platforms. Bear in mind, only devices that had this feature on their stock ROM will have this functionality in LineageOS ([#158337](https://review.lineageos.org/#/c/158337))
+* Torch on long-press has been added. This feature can be found in Settings > Buttons > Long-press power button for torch ([long-press-power-torch](https://review.lineageos.org/#/q/topic:long-press-power-torch))
+* CAF Rebase - some more information is below
+* Another build slave, courtesy of [DotSrc](http://dotsrc.org/)
+* Our in-house changelog viewer has been released. It's now available on the [download page](http://download.lineageos.org) ([#160901](https://review.lineageos.org/#/c/160901/))
+
+### CAF Rebase and February Security Patches
+When we started developing for Android 7.0 (not 7.1), we based our source code on CAF 7.0 (CAF we depend upon for almost all our Qualcomm based devices).  
+When Android 7.1 was released to AOSP, we merged this on top, but kept CAF 7.0 for HALs. Now CAF have updated their repositories to Android 7.1, we have rewritten history and rebased all merged commits on top of the CAF 7.1 base.
+This means:
+* much less time spent resolving merge conflicts
+* AOSP's February security patches (AKA r21) can be merged much more easily into LineageOS 14.1 (which is why we're later than usual for this month's security patches).  
+Before we did the huge rebase (happening over the past week), we created branches called `cm-14.1_prerebase` to make sure we had a known good copy to go back to, just in case the rebase did not go successfully.  
+We'll be working on incorporating the Feb security release into 13.0 builds in the near future.
+
+### Build roster
+Added 14.1 devices
+
+* Wileyfox Storm - kipper
+* Galaxy Tab S 8.4 (Wi-Fi) - klimtwifi
+* Galaxy Tab S 10.5 (Wi-Fi) - chagallwifi
+
+Added 13.0 devices
+
+* LeEco Le Pro3 - zl1
-- 
2.17.1

